modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -2849,8 +2849,19 @@ public CodegenParameter fromParameter(Parameter parameter, Set<String> imports)
             codegenParameter.vendorExtensions.putAll(parameter.getExtensions());
         }
 
-        if (parameter.getSchema() != null) {
-            Schema parameterSchema = ModelUtils.unaliasSchema(this.openAPI, parameter.getSchema());
+        Schema s;
+        if(parameter.getSchema() != null) {
+            s = parameter.getSchema();
+        } else if (parameter.getContent() != null) {
+            Content content = parameter.getContent();
+            MediaType mediaType = content.values().iterator().next();
```

Comment:
@zhemant thanks for the enhancement. Does it mean only the first medaType will be supported? If that's the case, what about logging a warning to inform the user?

Reply:
Yes, only first mediaType will be supported. I will push another commit with warning logging to inform the user.